DigitalExcellence/dex-frontend
# Bugfix/405 setup og tags

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

You can set up a localtunnel/NGROK network  and link this URL to any service that supports OG tags (facebook, twitter, discord, slack etc.). However, in this case the image will not be resolved, as it is linking to the dex.software site, which - at this point - does not have the appropriate header image yet (until the pull request is approved). 

When linked, you can check whether or not the description, URL and title work as expected. The header image can be checked by changing the link to the index.html file to whatever URL got generated and linking it to any service as listed above afterwards. 

example:
`  <meta property="og:image" content="https://dex.software/assets/images/dex-x.png">` 
becomes 
`  <meta property="og:image" content="https://rare-bird-82.loca.lt/assets/images/dex-x.png">`


## Link to issue

Closes: #405
